doozer/doozerlib/distgit.py

```diff
@@ -1286,6 +1289,61 @@ def update_build_db(self, success_flag, task_id=None, scratch=False):
                         traceback.print_exc()
                         self.logger.error(f'Unable to extract brew task information for {task_id}')
 
+    def get_installed_packages(self, image_pullspec) -> list:
+        bbii = BrewBuildImageInspector(self.runtime, image_pullspec)
+        installed_packages_dict = bbii.get_all_installed_package_build_dicts()
+        return [p['nvr'] for p in installed_packages_dict.values()]
+
+    def update_konflux_db(self, build_info, success, job_url='', scratch=False):
+        if scratch:
+            return
+
+        if not self.runtime.konflux_db:
+            self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
+            return
+
+        image_pullspec = build_info['extra']['image']['index']['pull'][0]
+
+        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
+        try:
+            source_repo, commitish = build_info['source'].split('#')
```

Comment:
I think this needs to be the `openshift-priv` URL and commitish we rebased from. Distgit commitish doesn't inform any decisions as far as I know. `openshift-priv`'s commit, however, allows us to say "do we have an image build for github.com/openshift-priv/component-repo-name @ commit deadbeef" . This is a necessary insight for ocp4_scan.
We can also capture the information you are presently storing here, but I would call it `rebase_repo_url` and `rebase_commitish`.

Reply:
Done in 2df1af0cb9b2524338761c80edf43da13216fcd3